specification/chap-dtrace-intermediate-format.tex

```diff
@@ -16,7 +16,10 @@ \section{The DIF Interpreter}
 within the \subroutine{dtrace_dif_emulate} subroutine, which has, as
 its first argument, a pointer to a DIF Object or
 \struct{dtrace_difo_t}.  Instructions are executed one at a time,
-until they are exhausted or an error causes interpretation to end.
+until they are exhausted or an error causes interpretation to end. The DIF
+objects are verified in the \subroutine{dtrace_difo_validate} subroutine and
+the DIF interpreter ignores any bounds checking within the
+\subroutine{dtrace_dif_emulate} subroutine for precisely this reason.
```

Comment:
"precisely this reason"  I think this needs to be reworded or you need to explain the reason.